R/chrome.R

```diff
@@ -172,6 +172,15 @@ chrome_print = function(
     if (missing(output) && !file.exists(input))
       output = xfun::with_ext(basename(gsub('[#?].*', '', url)), format)
     output2 = normalizePath(output, mustWork = FALSE)
+    # try to remove the file and throw a clear error if it still exists as it may be locked
+    if (!suppressWarnings(file.remove(output2)) && xfun::file_exists(output2)) {
+      stop(
+        sprintf("The file %s cannot be removed%s.",
+                sQuote(output),
+                if (format == "pdf") " (may be locked by a PDF reader)" else ""
+        )
+      )
```

Comment:
Just a small suggestion of a cosmetic change (feel free to accept or discard):

```suggestion
      stop(
        "The file '", output, "' cannot be overwritten",
        if (format == "pdf") " (may be locked by a PDF reader?)", "."
      )
```

Reply:
Oh indeed nicer. I missed that `stop()` can be used that way - that is why I find review useful ! thanks